spec/rubocop/cop/style/method_name_spec.rb

```diff
@@ -49,6 +49,16 @@ def +(other)
       END
     end
 
+    it 'accepts unary operator definitions' do
+      expect_no_offenses(<<-END.strip_indent)
```

Comment:
Definitely agree. 🙂 I tend to follow whatever the test file I'm working on is doing, but I don't mind changing this whole file to `RUBY`. 😊